analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -189,7 +189,6 @@ public static class Extensions
         }
 
         [DataTestMethod]
-        [DataRow("this?.InstanceMethod();")]
```

Comment:
This now has a full test method. Why isn't it just another DataRow of `Invocation_InstanceMethodCallDoesClearFieldOnThis` instead?